src/transformers/modeling_utils.py

```diff
@@ -1924,6 +1924,9 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
             load_in_8bit_skip_modules (`List[str]`, *optional*):
                 An explicit list of the modules that we do not want to convert in 8-bit. This is useful for models such
                 as Jukebox that has several heads in different places and not necessarily at the last position.
+            enable_memory_efficient_backward (`bool`, *optional*):
```

Comment:
As discussed on Felix's other PR, I am very much against adding new arguments to this method (which is the central piece of the Transformers API). Could this be added to the FP8 config we were discussing instead?

Reply:
Yes sure, another thing I was thinking about was to wrap all 8-bit arguments (`load_in_8bit_threshold`, etc.) inside `load_in_8bit_kwagrs` in case we merge this PR first. How does this sounds?

Reply:
We can't add a new kwarg to this heavily used method and remove it after, so we should really do the quantization_config first :-)

Reply:
Sounds good!

Reply:
There's a version of Linear8bitLt that supports 8-bit backward in all bitsandbytes models by default, without the flag
https://github.com/bigscience-workshop/petals/commit/088713912de15bbda62a8fab4ea345417b1b3b0f

The current memory_efficient_backward flag is that it slows down forward pass. The updated version (url above) would have the same inference time as the inference-optimized version.
Perhaps we can just patch the updated version to bitsandbytes? Then, transformers models will support 8bit backward automatically, we just need to write some tutorial / exmple for that.

Reply:
I think the solution to patch the efficient version into bitsandbytes is best. A pull request would be great, but I can also merge it manually the next time I work on bitsandbytes which might already be this weekend